### ISSUE TITLE
[New Feature][Network Scanner] Add a feature for scanning IPs

### ISSUE THREAD

**Issue by vinitshahdeo** [vinitshahdeo/PortScanner#11]
### **Feature to be added:** IP scanner 🔍 
> Used for scanning the networks that are connected to several computers.

Follow the similar code structure and add a new file (`ipScanner.py`) in the `root` directory. 

To make you understand why're we doing this:

![image](https://user-images.githubusercontent.com/20594326/84234620-337fdc80-ab12-11ea-9430-2944c82463ec.png)

<p align="center"><a href="https://my.gradwell.com/s/article/what-is-port-scanning">Image Source</a></p>

#### **DISCLAIMER:** This should be solely done for the learning purpose.

<sup>Feel free to post your thoughts here.</sup>

**Comment by ziyad00** [comment]
can you elaborate the feature

**Comment by vinitshahdeo** [comment]
@ziyad00 Within the given range of IPs, you have to scan the network to find the available hosts.

**Comment by vinitshahdeo** [comment]
@Kashish121 Take a look into this, can you raise a PR for the same by tomorrow EOD?

**Comment by Kashish121** [comment]
Will raise the PR soon, allow me some time.

**Comment by anandhakrishnanaji** [comment]
Hi there! I’m a first-time contributor and was hoping to help out with this issue. I noticed nobody was assigned to it, but if there’s already a solution in progress I’m happy to try helping out elsewhere. Thanks!

**Comment by vinitshahdeo** [comment]
@anandhakrishnanaji This is already done! @Kashish121, Can you please confirm if anything else is left for the same?

> You can check [this](https://github.com/vinitshahdeo/Water-Monitoring-System/issues/236) - **More `beginner-friendly` issues coming soon!**

**Comment by Kashish121** [comment]
@vinitshahdeo Yes this issue/feature has been resolved by me. It does not require any updates as of now.

**Comment by saisiddhant12** [comment]
@Kashish121 @vinitshahdeo can you mark this issue as closed?

**Comment by Meenasirisha** [comment]
Hi! I'm a first timer trying to contribute to open source. I would like to work on this

**Comment by Meenasirisha** [comment]
> @vinitshahdeo Yes this issue/feature has been resolved. It does not require any updates as of now.

So this is issue closed?
